scripts/npm/add-svgs.js

```diff
@@ -0,0 +1,106 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable no-console */
+
+const path = require('path');
+const fs = require('fs');
+const readline = require('readline');
+
+let readMoreFiles = true;
+const svgFiles = [];
+const errors = [];
+
+const processSvg = file => {
+  const filePath = path.dirname(file);
+  const fileName = path.basename(file);
+  const newFile = path.join('./packages/bpk-svgs/src/icons/', fileName);
+
+  fs.readFile(file, 'utf8', (err, data) => {
+    if (err) {
+      return console.log(err);
+    }
+    const pathRegex = /path d="(.*)"/gm;
+    const path = pathRegex.exec(data)[1];
+    console.log(`path`, path);
+    const result = `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="${path}"/></svg>`;
+
+    fs.writeFile(newFile, result, 'utf8', err2 => {
+      if (err2) return console.log(err2);
+      return null;
+    });
+    return null;
+  });
+};
+
+const questionAsync = (rl, question) => {
+  return new Promise((resolve, reject) => {
+    rl.question(question, answer => {
+      resolve(answer);
+    });
+  });
+};
+
+const readFilenames = async rl => {
+  const input = await questionAsync(
+    rl,
+    `SVGs: ${
+      svgFiles.length
+    }\nWhen you've added files, hit return. When you've got no more to add, just hit return with no files specified.\n`,
+  );
+  if (input === '') {
+    readMoreFiles = false;
+    return;
+  }
+  const fileNames = input.split(/(?<!\\)\s/);
+  fileNames.forEach(f => {
+    if (f !== '') {
+      svgFiles.push(f.split('\\').join(''));
+    }
+  });
+};
+
+console.log(
+  `What SVG(s) would you like to add?\nTIP 💡 You can drag file(s) here from finder instead of typing the paths yourself!\n`,
```

Comment:
```suggestion
  `What SVG(s) would you like to add?\nTIP 💡 You can drag file(s) here from Finder instead of typing the paths yourself!\n`,
```

😇